src/embind/embind.js

```diff
@@ -35,6 +35,9 @@ var LibraryEmbind = {
   $PureVirtualError__deps: ['$extendError'],
   $PureVirtualError: undefined,
   $GenericWireTypeSize: {{{ 2 * POINTER_SIZE }}},
+#if EMBIND_AOT
+  $InvokerFunctions: '<<< EMBIND_AOT_OUTPUT >>>',
```

Comment:
I think this might make the output invalid until the replacement occurs.. maybe thats fine?

Reply:
I could make it a comment again, either way if the EMBIND_AOT_OUTPUT is not included things will be broken.